students/paul_c/session03/list_lab.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Series 1
+print("\n*** Series 1 ***\n")
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruits)
+user_fruit = input("Please type in a new fruit > ")
+fruits.append(user_fruit)
+print(fruits)
+user_index = input("Please enter a number from 0 - 4 > ")
+user_selected = fruits[int(user_index)]
+print("You have selected fruit number", user_index, '"'+user_selected+'"', "from the list.")
+pineapples = ["Pineapples"]
+fruits = pineapples + fruits
+fruits.insert(0, "Papayas")
+for i in fruits:
+    if i.startswith("P|p"):
```

Comment:
`if i.lower().startswith("p"):`

Reply:
Cool, I didn't know you could do it that way. Thanks!